app/scripts/controllers/util/error.js

```diff
@@ -8,16 +8,18 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('ErrorController', function ($scope, $window) {
+  .controller('ErrorController', function ($scope, $location, $window, $uibModal, HomePagePreference, NotificationsService) {
     var params = URI(window.location.href).query(true);
     var error = params.error;
 
     switch(error) {
       case 'access_denied':
         $scope.errorMessage = "Access denied";
+        checkHomePagePreference(params.project_name);
```

Comment:
This would typically throw an error w/our `jshint` conf, using a function before it is declared.  Do we still enforce that rule?